js/custom.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function(){
+
+//user to be able to input text and save it
+$('form').submit(function(event){
```

Comment:
I'm not sure your form currently emits a submit event due to not having a submit button type specified, so don't think this event will fire.